lib/authenforce/middleware.go

```diff
@@ -1,21 +1,63 @@
 package authenforce
 
 import (
+	"context"
+	"encoding/json"
+	"io"
 	"net/http"
+	"sync/atomic"
 
+	"github.com/cuvva/cuvva-public-go/lib/cher"
 	"github.com/cuvva/cuvva-public-go/lib/crpc"
 	"github.com/wearemojo/mojo-public-go/lib/authparsing"
+	"github.com/wearemojo/mojo-public-go/lib/errgroup"
 )
 
-func (e Enforcer) CRPCMiddleware() crpc.MiddlewareFunc {
+func CRPCMiddleware(enforcers []Enforcer) crpc.MiddlewareFunc {
```

Comment:
Couple of things come to mind

1 - if we had another RPC package supported, would we abstract some of the logic into another function/place

2 - worth explaining somewhere the rules about "no more than one enforcer may handle" - e.g. on the enforcer, or wherever you first combine them together